yt/yt/ytlib/ya.make

```diff
@@ -742,6 +742,7 @@ PEERDIR(
     library/cpp/erasure
     library/cpp/iterator
     library/cpp/yt/backtrace/symbolizers/dwarf
+    library/cpp/cron_expression
```

Comment:
And yeah, dependency is probably misplaced. Depending on what you do, it will either go in server or directly into client.